addon/system/drag-listener.js

```diff
@@ -15,15 +15,15 @@ export default class {
     this._events = A();
   }
 
-  beginListening() {
+  beginListening(selector) {
```

Comment:
This is intentional, because there's issues where drag listeners don't get called consistently in browsers 😭 (The events will sometimes skip elements or repeat events on elements)

I _think_ the bug is with the `findListener` function.

Reply:
It seems that `evt.target.find(target)` should be removed in the `findListener` function. That will return `true` for all cases.

Reply:
Thanks very much for your reply ❤️ I have created a new pull request!